spark-extension/src/main/scala/org/apache/spark/sql/blaze/NativeConverters.scala

```diff
@@ -956,6 +957,9 @@ object NativeConverters extends Logging {
             .apply(precision, IntegerType) :: Literal.apply(scale, IntegerType) :: Nil
         buildExtScalarFunction("CheckOverflow", args, DecimalType(precision, scale))
 
+      case e: NormalizeNaNAndZero =>
```

Comment:
should we add `if e.dataType == IntegerType | LongType` constraints? as current implementation only supports floating numbers.